.mergify.yml

```diff
@@ -151,6 +99,48 @@ pull_request_rules:
         - files~=^\.mergify\.yml$
     actions:
       delete_head_branch:
+  - name: notify the backport policy
+    conditions:
+      - -label~=^backport
+      - base=master
+    actions:
+      comment:
+        message: |
+          This pull request does not have a backport label. Could you fix it @{{author}}? 🙏
+          To fixup this pull request, you need to add the backport labels for the needed
+          branches, such as:
+          * `backport-v./d./d./d` is the label to automatically backport to the `7./d` branch. `/d` is the digit
+
+          **NOTE**: `backport-skip` has been added to this pull request.
+      label:
+        add:
+          - backport-skip
+  - name: backport patches to 7.x branch
+    conditions:
+      - merged
+      - label=backport-v7.16.0
+    actions:
+      backport:
+        assignees:
+          - "{{ author }}"
+        branches:
+          - "7.x"
+        labels:
+          - "backport"
+        title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: backport patches to 7.14 branch
```

Comment:
Nit: Can we move this below 7.14? Like this it is 7.x, 7.15, 7.14 in the order

Reply:
I was tempted, but I think the release process adds the new branch to the end of the file. See https://github.com/elastic/beats/pull/27449

I thought an incremental order might be beneficial in this case. Otherwise, this will get mess anyway unless the automation for the release is adapted for this.

Reply:
SGTM